radio/src/mavlink_telem_camera.cpp

```diff
@@ -0,0 +1,306 @@
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDCMD_VIDEO_START_CAPTURE) {
+        RESETTASK(TASK_CAMERA, TASK_SENDCMD_VIDEO_START_CAPTURE);
+        generateCmdVideoStartCapture(_sysid, camera.compid);
+        set_request(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_CAPTURE_STATUS, 2);
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDCMD_VIDEO_STOP_CAPTURE) {
+        RESETTASK(TASK_CAMERA, TASK_SENDCMD_VIDEO_STOP_CAPTURE);
+        generateCmdVideoStopCapture(_sysid, camera.compid);
+        set_request(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_CAPTURE_STATUS, 2);
+        return true; //do only one per loop
+    }
+
+    return false;
+}
+
+
+bool MavlinkTelem::doTaskCameraLowPriority(void)
+{
+    if (!_task[TASK_CAMERA]) return false; // no task pending
+
+    if (_task[TASK_CAMERA] & TASK_SENDCMD_SET_CAMERA_VIDEO_MODE) {
+        RESETTASK(TASK_CAMERA, TASK_SENDCMD_SET_CAMERA_VIDEO_MODE);
+        generateCmdSetCameraMode(_sysid, camera.compid, CAMERA_MODE_VIDEO);
+        set_request(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_SETTINGS, 2);
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDCMD_SET_CAMERA_PHOTO_MODE) {
+        RESETTASK(TASK_CAMERA, TASK_SENDCMD_SET_CAMERA_PHOTO_MODE);
+        generateCmdSetCameraMode(_sysid, camera.compid, CAMERA_MODE_IMAGE);
+        set_request(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_SETTINGS, 2);
+        return true; //do only one per loop
+    }
+
+    // the sequence here defines the startup sequence
+    if (_task[TASK_CAMERA] & TASK_SENDREQUEST_CAMERA_INFORMATION) {
+        RESETTASK(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_INFORMATION);
+        generateCmdRequestCameraInformation(_sysid, camera.compid);
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDREQUEST_CAMERA_SETTINGS) {
+        RESETTASK(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_SETTINGS);
+        generateCmdRequestCameraSettings(_sysid, camera.compid);
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDREQUEST_CAMERA_CAPTURE_STATUS) {
+        RESETTASK(TASK_CAMERA, TASK_SENDREQUEST_CAMERA_CAPTURE_STATUS);
+        generateCmdRequestCameraCapturesStatus(_sysid, camera.compid);
+        return true; //do only one per loop
+    }
+    if (_task[TASK_CAMERA] & TASK_SENDREQUEST_STORAGE_INFORMATION) {
+        RESETTASK(TASK_CAMERA, TASK_SENDREQUEST_STORAGE_INFORMATION);
+        generateCmdRequestStorageInformation(_sysid, camera.compid);
+        return true; //do only one per loop
+    }
+
+    return false;
+}
+
+
+
+// -- Handle incoming MAVLink messages, which are for the Camera --
+
+void MavlinkTelem::handleMessageCamera(void)
+{
+    camera.is_receiving = MAVLINK_TELEM_RECEIVING_TIMEOUT; //we accept any msg from the camera to indicate it is alive
+
+    switch (_msg.msgid) {
+
+    case MAVLINK_MSG_ID_HEARTBEAT: {
```

Comment:
indentation needed

Reply:
all of "my" files are not formatted in a formatting styles I've seen in OpenTx. That's because I couldn't find a unique pattern followed in OpenTx and it hence wasn't clear what I should follow and hence just die "my thing". So, there will be probably 1000s of this comments.

ideally you could name me an example file which I could then follow

unless this should be indeed the only formatting comment, then I just will do 

pl advice

Reply:
https://github.com/opentx/opentx/wiki/OpenTX-Code-Style-Guide

Reply:
PERFECT !!!

many thx

(and mea culpa that I haven't realized that myself)

Reply:
did lots of formatting. Hopefully getting closer to the desired style now.

Reply:
Clear code formatting rules were documented by me 2 weeks ago. Eclipse time to time will not handle it all correctly if code is part of macro definition. I was facing same issue when trying to merge some code as well.